### PR TITLE
metrics/0001.lua passes

### PR DESCRIPTION
**Pull request by dominique-dymke**

### CODE REVIEW

File: platform/runner-extension/std/sys/src/main/kotlin/io/hamal/extension/std/sys/metric/GetMetricFunction.kt [hunk 1]
```diff
@@ -18,12 +18,9 @@ class GetMetricFunction(
...
         val data: MetricData = hubSdk.metric.get()
         val map = MapType()
-        map.set("time", data.time)
-        if (!data.events.isEmpty()) {
-            for (i in data.events) {
-                map.set(i.name.replace("::", "_"), i.value)
-            }
-        }
+        map["time"] = data.time
+        map["exec_completed"] = data.events.find { it.name == "exec::completed" }?.value ?: 0
```
Comment (dominique-dymke) [inline platform/runner-extension/std/sys/src/main/kotlin/io/hamal/extension/std/sys/metric/GetMetricFunction.kt:22]: Should always be set - either 0 or a value

File: platform/mono/src/integrationTest/resources/api/metric/0002.lua [hunk 2]
```diff
@@ -0,0 +1,18 @@
+--sys = require('sys')
```
Comment (dominique-dymke) [inline platform/mono/src/integrationTest/resources/api/metric/0002.lua:1]: test framework needs to be rewritten to accommodate this

File: platform/mono/src/integrationTest/kotlin/io/hamal/mono/api/BaseApiTest.kt [hunk 3]
```diff
@@ -110,6 +113,7 @@ abstract class BaseApiTest {
         execCmdRepository.clear()
         funcCmdRepository.clear()
         groupCmdRepository.clear()
+        metricRepository.clear()
```
Comment (dominique-dymke) [inline platform/mono/src/integrationTest/kotlin/io/hamal/mono/api/BaseApiTest.kt:116]: start clear on each test - otherwise test order would matter

**Review by jonsberg — APPROVED** [review]
merci

**Pull request merged**